dex/src/fees.rs

```diff
@@ -67,8 +78,13 @@ const fn rebate_tenth_of_bps(tenth_of_bps: u64) -> U64F64 {
 
 impl FeeTier {
     #[inline]
-    pub fn from_srm_and_msrm_balances(srm_held: u64, msrm_held: u64) -> FeeTier {
+    pub fn from_srm_and_msrm_balances(market: &Pubkey, srm_held: u64, msrm_held: u64) -> FeeTier {
         let one_srm = 1_000_000;
+
+        if market == &stable_markets::usdt_usdc::ID || market == &stable_markets::msol_sol::ID {
```

Comment:
Not an ideal implementation. But I dont think there's a much better way of doing this without breaking changes. Ideally the market state account would be flagged as stable on creation.